Orange/widgets/evaluate/owconfusionmatrix.py

```diff
@@ -426,11 +426,13 @@ def sum_item(value, border=""):
     def send_report(self):
         if self.results is not None and self.selected_learner:
             index = self.selected_learner[0]
+            cmatrix = confusion_matrix(self.results, index)
+            n= len(cmatrix)
```

Comment:
confusion matrix is a very small matrix as it contains the total number of different class values.
The maximum size of it would be 1000*1000 for a dataset of ImageNet and that we won't be using as it contains 1k different classes.
So In my opinion, it won't be a problem to calculate the confusion matrix again as it is very small dimensional and easy to construct.
What do you think?